app/controllers/completed_submissions_controller.rb

```diff
@@ -3,6 +3,8 @@ class CompletedSubmissionsController < ApplicationController
 	def index
 		@unprinted_submissions = MaterialSubmission.active.sort_by(&:id).reverse
 		@printed_submissions = MaterialSubmission.printed.sort_by(&:id).reverse
+		@dispatched_submissions = MaterialSubmission.dispatched.sort_by(&:id).reverse
+		@not_dispatched_submissions = MaterialSubmission.printed.not_dispatched.sort_by(&:id).reverse
```

Comment:
Instead of retrieving the records we want, then sorting by `id`, then reversing that, we really should be getting them from the db in the order we want:

```
MaterialSubmission.order(id: :desc).active
MaterialSubmission.order(id: :desc).printed
MaterialSubmission.order(id: :desc).dispatched
MaterialSubmission.order(id: :desc).printed.not_dispatched
```